docs/Developers/Development_Setup.rst

```diff
@@ -58,11 +58,23 @@ in your current directory, as coala only works for Python >= 3.4.4
     $ virtualenv coala-venv -p $(which python3)
 
 - Run coala-venv :
+    + If you are a Linux or Mac user:
+
     ::
 
         $ source coala-venv/bin/activate
         (coala-venv)$ deactivate # to exit the environment
 
+    + And if you are a Windows user:
+
+    ::
+
+        > coala-venv\Scripts\activate
+        (coala-venv)> deactivate # to exit the environment
+
+    If you run into any hiccups refer to
+    `this <https://virtualenv.pypa.io/en/stable/userguide/#activate-script>`__.
```

Comment:
never use `this` as a link anchor in technical documetnation.  Describe what it is.  When printed, the link may not be shown at all, and instead be a footnote.

Reply:
@jayvdb I have made the changes.